qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_SamplingMeasurement.py

```diff
@@ -0,0 +1,102 @@
+import warnings
+
```

Comment:
let's rename this module to `sampling_measurement` so that it's pep8.

Reply:
Done